src/lib/journey/callbacks/_utilities/policies.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
```

Comment:
This is the new policies component that will display an input's policies for validating the value. This way it's reusable between all the callbacks that use policies.